docs/documentation/development/bindings/profiles.md

```diff
@@ -89,4 +89,29 @@ Two methods have to be implemented to achieve this:
 
 ```java
 <item-type> MyItem { channel="<bindingID>:<thing-typeID>:MyThing:myChannel"[profile="MyScope:MyProfile"]}
-```
\ No newline at end of file
+```
+
+## Existing Profiles
+
+### OffsetProfile
+
+The `OffsetProfile` provides the possibility to adjust a value from a device before it arrives at the framework.
+An offset can be specified via the parameter `offset` which has to be a `QuantityType` or `DecimalType`.
+A positive offset is the amount of change from the device towards the framework and vice versa.
+
+```java
+Number MyItem { channel="<bindingID>:<thing-typeID>:MyThing:myChannel"[profile="offset", pattern="<transformFunction>:<valueFormat>"]}
```

Comment:
You are mixing here BNF notation (e.g. `<bindingID>`) with concrete examples (e.g. `MyThing`). Both make sense, but I wouldn't mix them in the same example.